tests/Person.spec.ts

```diff
@@ -44,7 +44,7 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
 
     await test.step(`Testdaten(Benutzer) löschen via API`, async () => {
       if (username) { // nur wenn der Testfall auch mind. einen Benutzer angelegt hat
-        await Header.button_logout.click();
+        await Header.logout();
```

Comment:
In Klasse.spec ist header kleingeschrieben. Bitte überall einheitlich verwenden.